web/src/typing.ts

```diff
@@ -16,11 +18,10 @@ import {user_settings} from "./user_settings";
 // when we are typing.  For the inbound side see typing_events.js.
 // See docs/subsystems/typing-indicators.md for more details.
 
-function send_typing_notification_ajax(data) {
-    channel.post({
+function send_typing_notification_ajax(data: string | JQuery.PlainObject | undefined): void {
```

Comment:
Do the callers pass these other types? This seems very likely wrong.

Reply:
Agreed ....a rather loose check ..will fixed this.